opentracing/utils.go

```diff
@@ -0,0 +1,27 @@
+package opentracing
+
+import (
+	"encoding/base64"
+	"fmt"
+	"net/http"
+)
+
+const (
+	OpenTracingContextIDHeader = "OpenTracing-Context-Id"
```

Comment:
I don't really have super-strong feelings about this... definitely see how what you have above makes sense, and also see how what's in RFC 1 and RFC 2 makes sense (as far as wire serialization is concerned). Do you think one is clearly better than the other? If so, can you help me get "off the fence"?